android/gradle.properties

```diff
@@ -57,3 +57,9 @@ disableFrameProcessors=true
 android.nonTransitiveRClass=false
 
 org.gradle.parallel=true
+
+# Specify the package name for the patched react-native artifacts.
+patchedArtifacts.packageName=react-standalone
+
+# Change this property to false if you want to use prebuilt react-native artifacts.
+patchedArtifacts.forceBuildFromSource=true
```

Comment:
@roryabraham I've disabled prebuilt artifacts by default.

I suggest we merge this PR first and then I can test it internally within the team. Once we're confident about the entire flow, we can create a PR to enable prebuilt artifacts by default.

To use prebuilt artifacts, users will need to configure GitHub CLI. The plugin will display a log about misconfigured CLI and fall back to building from source.